### PR TITLE
adds visual effects to plant helper artifacts

### PR DESCRIPTION
**Pull request by zjdtmkhzt**
## About the PR 
Adds a visual effect to plant helper artifacts.
## Why's this needed? 
It's nice and it shows that they actually do things, before it was kinda like "hi, this will help your plants grow, maybe, kind of?"
Also, it makes it easier to see the radius, which varies wildly from artifact to artifact.
## Changelog

```changelog
(u)zjdtmkhzt
(+)Added visual effect to plant helper artifacts.
```

https://user-images.githubusercontent.com/35579460/109394334-c540ab80-7926-11eb-9356-e7902cddd649.mp4

### CODE REVIEW

**Comment by ZeWaka** [comment]
What do the colors mean

**Comment by zjdtmkhzt** [comment]
They show what the helper is doing, blue means it is adding water, red means it is removing it, green means growth, orangeish is supposed to be health, weedkilling is red too, when it mutates, it's supposed to be really dark.

File: code/obj/artifacts/artifact_machines/plant_helper.dm [hunk 1]
```diff
@@ -32,21 +32,54 @@
...
+						var/globalImageKey = "linetest[rand(0,INFINITY)]"
+						R.lineImage.color = lineColor
+						R.lineImage.alpha = 0
+						R.lineImage.plane = PLANE_SELFILLUM
+						R.lineImage.blend_mode = BLEND_ADD
+						addGlobalImage(R.lineImage, globalImageKey)
+						animate(R.lineImage, alpha = 64, time = 1 SECOND)
+						sleep(1 SECONDS)
+						animate(R.lineImage, alpha = 0, time = 1 SECOND)
+						sleep(1 SECONDS)
```
Comment (ZeWaka) [inline code/obj/artifacts/artifact_machines/plant_helper.dm:83]: can't we use animate timers here instead or something

Comment (zjdtmkhzt) [reply code/obj/artifacts/artifact_machines/plant_helper.dm:83]: Oh, yeah, I was thinking that while testing, but then I forgot about it! I'll try to do it with timers!

**Pull request merged**